utils/extractor/src/index.ts

```diff
@@ -1,8 +1,10 @@
-import api, { Flatfile } from '@flatfile/api'
+import { Flatfile, FlatfileClient } from '@flatfile/api'
 import type { FlatfileListener } from '@flatfile/listener'
 import { createAllRecords, slugify } from '@flatfile/util-common'
 import { getFileBuffer } from '@flatfile/util-file-buffer'
 
+const api = new FlatfileClient()
```

Comment:
Do we want to call this inside the Extractor function so it's scoped to that instance or is it more performant to put here now that we have the token fetching more dynamic?

Reply:
I kept the FlatfileClient initialization at the module level to maintain consistency with our other files and to leverage its design. The FlatfileClient is designed for efficient reuse with lazy-loaded resources and dynamic token handling. Creating it inside each function that needs it would prevent resource caching and create unnecessary client instances without providing any meaningful benefits.